### DISCUSSION TITLE
Swiper Coverflow effect's center slide active by default

### DISCUSSION THREAD

**Discussion by omprakash-prajapati** [nolimits4web/swiper#6964] [Q&A]
I have 8 slides in the slider and by default left side is blank and the slides are on the right side, But by default, I want to keep the center slide active. How to do that?

Default slides view
![image](https://github.com/nolimits4web/swiper/assets/102156642/914fbca6-6b82-4c44-b2da-6592d5c92e77)

Desired default slides view
![image](https://github.com/nolimits4web/swiper/assets/102156642/e3dde1ff-892d-437d-b557-c485900b0fc5)

**Comment by Arhell** [comment]
Hi
Try this https://codepen.io/mlegait/pen/KoEJEK
https://swiperjs.com/swiper-api#param-initialSlide

**Reply by abdulfattahaud** [reply]
That doesn't change anything
